src/avatar.cpp

```diff
@@ -849,7 +852,7 @@ nc_color avatar::basic_symbol_color() const
     if( has_effect( effect_boomered ) ) {
         return c_pink;
     }
-    if( has_active_mutation( trait_SHELL2 ) ) {
+    if( in_shell ) { // NOLINT(bugprone-branch-clone)
```

Comment:
Why do you need the nolint? I don't see a reason for it to be here.

Reply:
I saw it in another situation where `if( in_shell )` was used and so I copypasted it in case it's needed for something arcane that would break otherwise.

Reply:
Ah, okay. it's not necessary here.
```suggestion
    if( in_shell ) {
```

Reply:
Done!